arctic/store/bson_store.py

```diff
@@ -56,36 +56,90 @@ def find(self, *args, **kwargs):
         return self._collection.find(*args, **kwargs)
 
     @mongo_retry
-    def insert_one(self, value):
+    def insert_one(self, document, **kwargs):
         """
         See http://api.mongodb.com/python/current/api/pymongo/collection.html#pymongo.collection.Collection.insert_one
         """
         self._arctic_lib.check_quota()
-        return self._collection.insert_one(value)
+        return self._collection.insert_one(document, **kwargs)
 
     @mongo_retry
-    def insert_many(self, values):
+    def insert_many(self, documents, **kwargs):
         """
         See http://api.mongodb.com/python/current/api/pymongo/collection.html#pymongo.collection.Collection.insert_many
         """
         self._arctic_lib.check_quota()
-        return self._collection.insert_many(values)
+        return self._collection.insert_many(documents, **kwargs)
 
     @mongo_retry
-    def delete_one(self, query):
+    def delete_one(self, filter_, **kwargs):
```

Comment:
Just `filter` to match the underlying API? Otherwise you can end up in a funny state whereby the kwargs contains filter but you're massing it in as the first arg